assets/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "multiselect-static",
+  "version": "1.0.0",
+  "dependencies": {
+    "grunt": "^0.4.5",
```

Comment:
These dependencies are very out of date. Could you update them to latest while here? (see stable Preside package.json for example).